Dockerfile

```diff
@@ -15,12 +15,33 @@ RUN poetry config virtualenvs.create false --local
 RUN poetry install
 
 # PROD IMAGE
-    FROM base as production
-    COPY todo_app todo_app 
-    EXPOSE 5000
-    ENTRYPOINT [ "gunicorn","--bind","0.0.0.0:5000","todo_app.app:app"]
+FROM base as production
+RUN pip install gunicorn
+COPY todo_app todo_app 
+EXPOSE 5000
+ENTRYPOINT [ "gunicorn","--bind","0.0.0.0:5000","todo_app.app:wsgi_c"]
 
 # DEV IMAGE
-    FROM base as development
-    EXPOSE 5100
-    ENTRYPOINT [ "flask","run","--host","0.0.0.0"]
\ No newline at end of file
+FROM base as development
+EXPOSE 5100
+ENTRYPOINT [ "flask","run","--host","0.0.0.0"]
+
+# TEST IMAGE
+FROM base as test
+# INSTALL CHROME
+RUN curl -sSL\
+ https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb \
+ -o chrome.deb && apt-get update && apt-get install ./chrome.deb -y &&\
+ rm ./chrome.deb
+# INSTALL CHROMIUM WEBDRIVER
+RUN LATEST=`curl -sSL \
+https://chromedriver.storage.googleapis.com/LATEST_RELEASE` && \
+echo "Installing chromium webdriver version ${LATEST}" && curl -sSL \
+https://chromedriver.storage.googleapis.com/${LATEST}/chromedriver_linux64.zip \
+-o chromedriver_linux64.zip &&  apt-get install unzip -y &&\
+unzip ./chromedriver_linux64.zip
+COPY todo_app todo_app
+COPY tests tests
+COPY tests_e2e tests_e2e
+ENV PYTHONPATH "${PYTHONPATH}:/app"
```

Comment:
Worth noting that you can skip this line if you add an empty `__init__.py` file to your tests/tests_e2e folders which is good practice to help Python determine modules

Reply:
Thanks - I will add this to my to-do list, will address as a stretch goal later in the program.